python/pyspark/sql/pandas/utils.py

```diff
@@ -47,11 +49,13 @@ def require_minimum_pyarrow_version():
     try:
         import pyarrow
         have_arrow = True
-    except ImportError:
+    except ImportError as error:
         have_arrow = False
+        raised_error = error
     if not have_arrow:
         raise ImportError("PyArrow >= %s must be installed; however, "
-                          "it was not found." % minimum_pyarrow_version)
+                          "it was not found or an error was thrown by pyarrow"
```

Comment:
ditto. A space? `pyarrow"` -> `pyarrow "`?

Reply:
Addressed in https://github.com/apache/spark/pull/31902/commits/30542f345f739a63c67c876b4b87f7a0240f228e